client/src/main/com/sinch/sdk/domains/conversation/models/credentials/ConversationChannelWhatsApp.java

```diff
@@ -0,0 +1,42 @@
+package com.sinch.sdk.domains.conversation.models.credentials;
+
+import com.sinch.sdk.domains.conversation.models.ChannelType;
+
+public class ConversationChannelWhatsApp extends ConversationChannel {
```

Comment:
Because of incoming next OAS release, OAS files was not update to reflect it: just at business level.
Ideally such inheritance should be highlighted at OAS level

Reply:
Ok, let's see what will come to us in the "official" documentation file